doc/md/ddl/intro.md

```diff
@@ -160,7 +157,7 @@ show "seinfeld" {
   var test struct {
     Shows []*Show `spec:"show"`
   }
-  err := Unmarshal([]byte(f), &test)
+  err := EvalBytes([]byte(f), &test, nil)
```

Comment:
Again, why do you add the type here and not just leave it as Bytes? It's not like we have a few ways to Eval, no?